src/EditorFeatures/Core/Remote/SolutionChecksumUpdater.cs

```diff
@@ -64,19 +62,16 @@ protected override async Task ExecuteAsync()
                 _currentToken = null;
             }
 
-            // wait for global operation to finish
-            await GlobalOperationTask.ConfigureAwait(false);
```

Comment:
intentionally removed (will make more sense later).  we are only called into ExecuteAsync when we're not paused due to a global operation being in flight.